Telemetry/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.1.0] - 2022-10-21
+### Added
+- Added uploadLogs call
```

Comment:
Pls add description for the API. We could include the summary from Telemetry.json.